scanpy/preprocessing/_simple.py

```diff
@@ -289,15 +300,15 @@ def filter_genes(
 
 @singledispatch
 def log1p(
-    X: AnnData | np.ndarray | spmatrix,
+    data: AnnData | np.ndarray | spmatrix,
```

Comment:
I think you need to do this because this is now a part of the testing suite, but I am not sure this is such a good idea with breaking changes, unless I am missing something.  I have definitely used `X=` syntax before for no other reason than it exists

Reply:
Sounds good!